### ISSUE TITLE
Plugin don't work on server start and need reload to work

### ISSUE THREAD

**Issue by GamerOfEnder** [AcmeProject/WildernessTp#55]
Hey, i switched my server to paper from spigot and i still have this issue:
the is considered loaded by the server but when i try /rtp or any command it don't work and if i reload it with plugman it works.
Could you guys please help me ? I don't know what you need so i'll send the latest log
[latest.log](https://github.com/AcmeProject/WildernessTp/files/3908281/latest.log)

**Comment by RoboMWM** [comment]
Are you sure you have permission to run the command?

**Comment by RoboMWM** [comment]
try without any other plugins

**Comment by GamerOfEnder** [comment]
Yes i have all the permissions, i tried as op too but still don't load at start. I don't tried without others plugins.

**Comment by GamerOfEnder** [comment]
```ActionHealth*, AdvancedBan*, AdvancedHelp, ArmorStandTools, BKCommonLib, ChatManager, ChestCommands, ClearLag, CommandSpy*, CoreProtect, Essentials, EssentialsAntiBuild, EssentialsChat, EssentialsGeoIP, EssentialsSpawn, HolographicDisplays, HoloPlots, iDisguise, KillerMoney*, LuckPerms, Multiverse-Core, Multiverse-NetherPortals, MySkin, PerWorldInventory, PlotSquared, PlugMan, ProtocolLib, SuperVanish, TAB, TitleManager*, TreysCommandBlocker*, UltimateFreeze*, Vault, Wild, WorldEdit``` here is the list of all my plugins

**Comment by RoboMWM** [comment]
Try without other plugins, I'm guessing something may be messing with the command map or something

**Comment by RoboMWM** [comment]
do your worlds load in after server start or something?
